SubmittingPatches-backports.rst

```diff
@@ -17,7 +17,7 @@ General principles
 ------------------
 
 To help the people who will review your backport, please state either in the
-backport PR, or in the backport tracker issue, or in the master tracker issue:
+backport PR, or in the backport tracker issue, or in the main tracker issue:
```

Comment:
To me `main` is even more generic than `master`, so I might make this `main branch tracker issue`.  I'd also be tempted to backtick every instance of `main` in the file but that'd be a lot of work.

Reply:
there are several places where the term "main tracker issue" appears. 
not clear if this is the "tracker issue of the main branch" or the "main tracker issue" (see line 140).
I prefer to keep this fix minimal.

Reply:
made the changes